plugins/ProfileExtender/class.profileextender.plugin.php

```diff
@@ -236,6 +236,8 @@ private function profileFields($Sender) {
 
         // Get user-specific data
         $this->UserFields = Gdn::userModel()->getMeta($Sender->Form->getValue('UserID'), 'Profile.%', 'Profile.');
+        $this->EventArguments['userFields'] = &$this->UserFields ?? '';
+        $this->fireEvent('modifyUserFields');
```

Comment:
The event is normally named on the current "event" happening, by that I mean what we are currently doing when firing it. This event name sounds more like what we are doing in the hook.

Reply:
beforeGetProfileFields, better? @topcheese13

Reply:
Personally I do like it better.